distribution/openrct2.d.ts

```diff
@@ -1234,9 +1235,7 @@ declare global {
         readonly endBank: TrackBanking;
 
         /**
-         * The length of the segment in RCT track length units.
-         *
-         * *1 metre = 1 / (2 ^ 16)*
+         * The rough length of the segment.
```

Comment:
The only user of this value is when calculating the circuit length to determine the number of powered vehicles allowed on some ride types (like log flume). It is in no way accurate to the length of the piece - that data is the subposition data.